webcompat/static/css/development/components/icon-font.css

```diff
@@ -13,8 +13,16 @@
 .wc-Icon {
   display: inline-block;
   vertical-align: middle;
-  font: normal normal 400 normal 1em/1 webcompat;
+
+  /* to prevent a webcompat issue, we do not use the short for font form here right now
+  webkit has a css2 implementation in 10.1.1. regarding to that spec */
```

Comment:
@zoepage wrote:
> Yes, it is that one

OK, can we update the comment then. It's not accurate to say Safari implements CSS2 for `font`  -- it's just a regression that we need to work around. Thanks!